pvanalytics/quality/irradiance.py

```diff
@@ -238,8 +255,10 @@ def check_irradiance_limits_qcrad(solar_zenith, dni_extra, ghi=None, dhi=None,
        Science Journal 2, pp. 23-37, 2008.
 
     """
-    if not limits:
-        limits = QCRAD_LIMITS
+    if limits == 'physical':
+        limits = BSRN_LIMITS_PHYSICAL
+    elif limits == 'extreme':
+        limits = BSRN_LIMITS_EXTREME
```

Comment:
These lines could also be deleted, and thus pass the limits parameter directly to the individual QC functions. The functionality would be the same and I think it would be somewhat more intuitive.